core/src/main/java/com/oath/oak/OakWriteBuffer.java

```diff
@@ -23,7 +23,6 @@
      * @return This buffer
      * @throws IndexOutOfBoundsException If <tt>index</tt> is negative
      *                                   or not smaller than the buffer's limit
```

Comment:
I am adding it here, because there is no other relevant change line I can use. In the comment explaining the interface classes OakWriteBuffer and OakReadBuffer, please add the explanation of being "attached". If our README won't be read at least in the comment also say that these buffers can not be used outside of their scope method.

Reply:
Do you suggest pasting here the attached buffers' description from the readme?